tf-modules/cumulus-rds-tf/outputs.tf

```diff
@@ -6,6 +6,10 @@ output "rds_endpoint" {
   value = aws_rds_cluster.cumulus.endpoint
 }
 
+output "rds_reader_endpoint" {
+  value = aws_rds_cluster.cumulus.reader_endpoint
+}
```

Comment:
Do we have documentation we're providing on RDS Reader endpoints?

Reply:
Not yet. I added this output in anticipation of a future ticket which will utilize the reader endpoints. There is also a separate v2 documentation update ticket, Ref: https://bugs.earthdata.nasa.gov/browse/CUMULUS-3671

Reply:
Ah ok, thanks.